### DISCUSSION TITLE
I have another modpack crash :(

### DISCUSSION THREAD

**Discussion by Zazamu333** [FabricMC/fabric-api#3405] [Player support]
### I hace another modpack game crash. :(
###Log
```txt
https://mclo.gs/awhDewC
```

### Description

So, I launch the game with no problems, the issue comes when I try to create a new world, which is when it crashes. 
I have 364 mods in minecraft 1.20.1 (fabric 0.14.24)
Anyone that could help me please ?

**Comment by Linguardium** [comment]
mod All the Trims is failing to load a proper json file to configure trims.

**Reply by Zazamu333** [reply]
Thanks for answer :D so I will delate it and replace it with another tirms mod.
